en/0x10-V1-Inventory.md

```diff
@@ -17,13 +17,13 @@ transparency within the organization, promotes software standardization and reus
 
 | # | Description | L1 | L2 | L3 |
 | :---: | :--- | :---: | :---: | :---: |
-| **1.1** | All components and their versions are known at completion of a build | ✓ | ✓ | ✓ |
-| **1.2** | Package managers are used to manage all third-party binary components | ✓ | ✓ | ✓ |
+| **1.1** | All constituent components and their versions are known at completion of a build | ✓ | ✓ | ✓ |
```

Comment:
I added constituent because it wasn't clear what you are referring to. It is the constituent or the generated component or both. I have nevertheless reverted.